packages/docusaurus-plugin-content-docs/src/markdown/__tests__/__fixtures__/docs/doc2.md

```diff
@@ -9,4 +9,4 @@
 - [doc1](doc1.md)
 - [doc2](./doc2.md)
 
-- [doc-localized](./doc-localized.md)
+- [doc-localized](/doc-localized.md)
```

Comment:
This change is intended. This file is actually not in the localized directory, so relative links shouldn't magically point to the `fr` localized directory. Changing it to an absolute link will cause the `fr` directory to be matched in priority.